backend/users/auth.py

```diff
@@ -13,9 +13,6 @@ class CustomUserBackend(ModelBackend):
 
     user_models = (Faculty, ResearchScholar, Student)  # order is important
 
-    def authenticate(self, *args, **kwargs):
-        return self.downcast_user_type(super().authenticate(*args, **kwargs))
```

Comment:
Change the doc string according to this change,

Reply:
Which docstring?

Reply:
nvm, I interpreted it wrongly. Doc string is correct.